backend/main.go

```diff
@@ -16,6 +16,8 @@ func main() {
 	// test route to add data to firestore
 	r.HandleFunc("/add", AddData).Methods("POST")
 	r.HandleFunc("/get/{id}", GetData).Methods("GET")
+	r.HandleFunc("/delete/{id}", DeleteData).Methods("DELETE")
+	r.HandleFunc("/update/{id}", UpdateUser).Methods("PATCH")
```

Comment:
Apologies if I'm overstepping here since it's a draft PR but I'm assuming these aren't the final routes?

Reply:
yeah these are just for testing for now